modules/claims_api/app/services/claims_api/unsynchronized_evss_claims_service.rb

```diff
@@ -28,6 +28,14 @@ def update_from_remote(evss_id)
       create_claim(evss_id, :data, raw_claim)
     end
 
+    def veteran
+      @veteran ||= Veteran.new(@user)
+    end
+
+    def poa
+      @veteran.poa
```

Comment:
do you want to call the method or the instance variable here? I _think_ calling the instance variable could return it unset if the method isn't called first

Reply:
Also, is this method now power_of_attorney on the veteran model as opposed to `poa`?

Reply:
Done